cli/types/cypress.d.ts

```diff
@@ -5567,6 +5567,12 @@ declare namespace Cypress {
      * @see https://on.cypress.io/catalog-of-events#App-Events
      */
     (action: 'command:end', fn: (command: CommandQueue) => void): Cypress
+    /**
+     * Fires when a command is skipped, namely the `should` command
```

Comment:
```suggestion
     * Fires when a command is skipped, namely the `should` command.
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/19601/commits/5ac4525c211aded3fde2db7ce52d7d0b1e405938